cmd/config_init.go

```diff
@@ -28,19 +26,18 @@ to the defaults when creating new users and you don't
 override the options.`,
 	Args: cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
+		log.Println(cfgFile)
 		databasePath := v.GetString("database")
 		if _, err := os.Stat(databasePath); err == nil {
-			panic(errors.New(databasePath + " already exists"))
+			log.Fatal(databasePath + " already exists")
 		}
 
 		defaults := settings.UserDefaults{}
 		getUserDefaults(cmd, &defaults, true)
 		authMethod, auther := getAuthentication(cmd)
 
-		db, err := storm.Open(databasePath)
-		checkErr(err)
+		db, st := getStorage()
```

Comment:
I'll revert this. Take a look at `getStorage` code and you'll see why it can't be used on `config init` specifically. Wait for a commit here soon 🛩